eng/restore/binplacePackages/Configurations.props

```diff
@@ -12,6 +12,7 @@
       netstandard2.0;
       netstandard;
       netcoreapp2.0;
+      netcoreapp3.0;
```

Comment:
why is this still required?

Reply:
I think we don't require netcoreapp3.0 but require netcoreapp for apicompact between netcoreapp and netstandard. I am just verifying.

Reply:
We should not restore anything here just for the sake of ApiCompat.

Reply:
This would result in adding these types ApiCompatBaseline.Txt files which is also not good.
We should not use apiCompatBaseline.txt files to work around stuff. 
The potential harm here is people adding a reference to something that they shouldn't. but this can be caught in reviews.
However, I don't feel strongly either way, just wanted to bring up why adding netcoreapp could be good.

Reply:
We still require netcoreapp3.0 because we still build the src with config netcoreapp3.0 and it will otherwise pick a netstandard dll for reverseApiCompat

Reply:
@ericstj I'm still unsure about binplacing restored netcoreapp5.0 assets here. I don't think that's a good idea but I'm happy to be proven wrong. When I created that depproj it was never intended to be used for netcoreapp.

Reply:
Resolved offline